json-c.sym

```diff
@@ -16,8 +16,10 @@ JSONC_PRIVATE {
     array_list_del_idx;
     array_list_free;
     array_list_new;
+    array_list_insert_idx;
```

Comment:
Please do not add anything to JSONC_PRIVATE.  Unlike the previously existing symbols, if these new ones are internal-only, they shouldn't be exported at all.

Reply:
ack